models/db.go

```diff
@@ -0,0 +1,20 @@
+package models
```

Comment:
It might be a good idea to keep database models and database connectivity code separately.

Reply:
I will split that later.